source/Application/Component/Widget/CookieNote.php

```diff
@@ -55,10 +55,6 @@ public function render()
      */
     public function isEnabled()
     {
-        if ($this->getConfig()->getConfigParam('blShowCookiesNotification')) {
-            return true;
-        }
-
-        return false;
+        return $this->getConfig()->getConfigParam('blShowCookiesNotification');
```

Comment:
It would be best if method would return only `true` or `false`. In this case it returns `null` too.